test/common/__snapshots__/graph.js.snap

```diff
@@ -20,36 +20,48 @@ exports[`graph #visitor should visit a model manager 1`] = `
    \`org.acme.hr.base@1.0.0.Level\` --> \`concerto@1.0.0.Concept\`
    \`org.acme.hr.base@1.0.0.Time\`
    \`org.acme.hr.base@1.0.0.SSN\`
+   \`org.acme.hr@1.0.0.pii\`
+   \`org.acme.hr@1.0.0.pii\` --> \`org.acme.hr@1.0.0.Category\`
+   \`concerto.decorator@1.0.0.Decorator\`
+   \`concerto.decorator@1.0.0.Decorator\` <--> \`org.acme.hr@1.0.0.pii\`
```

Comment:
Why the double arrow here?

Reply:
`pii extends Decorator`

This would mean a node connecting `pii` to `Decorator` and vice versa.

Reply:
Why does `Decorator` depend on `pii`?

Reply:
The issue seems to be that we add an edge from the `SuperType` to it's dependency

```
if(!parameters.createDependencyGraph && classDeclaration.getSuperType() !== 'concerto@1.0.0.Concept') {
    parameters.graph.addVertex(classDeclaration.getSuperType());
    parameters.graph.addEdge(classDeclaration.getSuperType(), classDeclaration.getFullyQualifiedName());
}
 ```

Reply:
Ah, I see. I guess this ensures that if we get the graph for X we will include everything that X depends on, and everything that depends on X. You can see how this parameter should be used very carefully!